store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9758,4 +9758,23 @@ TODO: delete them permanently from here
   <desc>URL of the event storage backend to be queried for event analytics</desc>
 </attr>
 
+<attr id="3046" name="zimbraAffinityEventLoggingEnabled" type="boolean" cardinality="single" optionalIn="account,cos" flags="accountInherited" callback="EventLoggerCallback" since="8.8.6">
+  <desc>If TRUE, incoming emails will cause an AFFINITY event to be logged for each recipient. This allows for more accurate "related contacts" results, at the cost of higher index usage</desc>
+  <defaultCOSValue>TRUE</defaultCOSValue>
+</attr>
+
+<attr id="3047" name="zimbraRelatedContactsMaxAge" type="duration" cardinality="single" optionalIn="account,cos" flags="accountInherited" since="8.8.6">
+  <desc>The time window for which related contacts are calculated; emails older than this will not affect the contact affinity calculation</desc>
+  <defaultCOSValue>365d</defaultCOSValue>
+</attr>
+
+<attr id="3048" name="zimbraRelatedContactsMinCooccurrenceCount" type="integer" min="1" cardinality="single" optionalIn="account,cos" flags="accountInherited" since="8.8.6">
```

Comment:
Typo - MonCoocc....  Suggest **zimbraRelatedContactsMinConcurrenceCount**

Reply:
"Cooccurrence" is somewhat of an awkward word to type...

Reply:
Ah.  Ok, so that was intentional.